Feedback/models.py

```diff
@@ -30,14 +30,27 @@ class Section(models.Model):
 	section_name = models.CharField(max_length=3)
 	year = models.IntegerField(validators=[
 		MaxValueValidator(4),
-		MinValueValidator(1)
-		])
+		MinValueValidator(1) ])
 	branch = models.CharField(max_length=20)
 	shift = models.IntegerField(validators=[
 		MaxValueValidator(2),
-		MinValueValidator(1)
-		])
+		MinValueValidator(1) ])
 	subjects = models.ManyToManyField(TeacherSubjectMapping)
 
 	def __str__(self):
 		return self.section_name
+
+
+class Feedback(models.Model):
+	teacher = models.ForeignKey(Teacher, on_delete=models.CASCADE)
+	section = models.ForeignKey(Section, on_delete=models.CASCADE, null=True)
+	Punctuality = models.IntegerField(blank=False)
+	Subject_knowledge = models.IntegerField(blank=False)
+	Behaviour = models.IntegerField(blank=False)
+	Method_of_teaching = models.IntegerField(blank=False)
+	Audibility = models.IntegerField(blank=False)
+	Syllabus_coverage = models.IntegerField(blank=False)
+
+	def __str__(self):
+		return f"{str(self.teacher)}-{str(self.section)}"
+
```

Comment:
There is an extra line at the end of file. If possible kindly remove it. 
And yes, it would show 2 blank lines in your text editor. You just have to remove one.